test/test_create_database.py

```diff
@@ -16,7 +16,7 @@ def test_postgres_database_schema_exists(self):
         db_schema = 'partysvc'
 
         with patch('run.create_engine'), patch('run.scoped_session'), patch('run.command.upgrade') as upgrade:
-            create_database(db_connection, db_schema)
+            create_database(db_connection, db_schema, 5, 10, -1)
```

Comment:
Are these numbers meant to be configurable by the env vars?

Reply:
Keeping them hard coded here is fine as the tests focus on the alembic stuff.